tools/Config.mk

```diff
@@ -458,7 +458,8 @@ define CLEAN
 endef
 else
 define CLEAN
-	$(Q) rm -f *$(OBJEXT) *$(LIBEXT) *~ .*.swp
+	$(call DELFILE, $(wildcard $(foreach obj, $(OBJS), $(addsuffix /$(obj), $(subst :, ,$(VPATH))))))
```

Comment:
My idea was not to use VPATH for this. The $(OBJ) should have a path relative to where this CLEAN macro is executed if I understand correctly.